frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -94,7 +94,8 @@ const getEventSourceResource = (
   let eventSourceResources = [];
   const ownerUid = _.get(dc, ['metadata', 'ownerReferences', '0', 'uid'], null);
   if (ownerUid) {
```

Comment:
Wouldn't be easier to just not do this if-statement?

Reply:
Thats... not what I meant. I meant just not do the logic in the if-statement when you didn't have the data for it.

You were only doing a filter.

Reply:
you mean not to do `filter` if data is not there, default value id `[]` so shall I have a check ?